Q/Qt6Charts/build_tarballs.jl

```diff
@@ -0,0 +1,88 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Qt6Charts"
+version = v"6.7.1"
+
+# Collection of sources required to build qt6
+sources = [
+    ArchiveSource("https://download.qt.io/official_releases/qt/$(version.major).$(version.minor)/$version/submodules/qtcharts-everywhere-src-$version.tar.xz",
+                  "c236875ab3c812f0d59bad81a783095ae50932c7bea23cb81cd55437083d5d8c"),
+    ArchiveSource("https://github.com/roblabla/MacOSX-SDKs/releases/download/13.3/MacOSX13.3.sdk.tar.xz",
+                  "e5d0f958a079106234b3a840f93653308a76d3dcea02d3aa8f2841f8df33050c"),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+
+mkdir build
+cd build/
+qtsrcdir=`ls -d ../qtcharts-*`
+
+case "$bb_full_target" in
+
+    x86_64-linux-musl-libgfortran5-cxx11)
+        cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_FIND_ROOT_PATH=$prefix -DCMAKE_BUILD_TYPE=Release $qtsrcdir
+    ;;
+
+    *apple-darwin*)
+        apple_sdk_root=$WORKSPACE/srcdir/MacOSX13.3.sdk
+        sed -i "s!/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root!$apple_sdk_root!" $CMAKE_TARGET_TOOLCHAIN
+        cmake -G Ninja -DQT_HOST_PATH=$host_prefix \
+            -DPython_ROOT_DIR=/usr \
+            -DCMAKE_INSTALL_PREFIX=${prefix} \
+            -DCMAKE_PREFIX_PATH=$host_prefix \
+            -DCMAKE_FIND_ROOT_PATH=$prefix \
+            -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+            -DCMAKE_SYSROOT=$apple_sdk_root -DCMAKE_FRAMEWORK_PATH=$apple_sdk_root/System/Library/Frameworks -DCMAKE_OSX_DEPLOYMENT_TARGET=11 \
+            -DQT_NO_APPLE_SDK_AND_XCODE_CHECK=ON \
+            -DCMAKE_BUILD_TYPE=Release \
+        $qtsrcdir
+    ;;
+
+    *)
+        cmake -DQT_HOST_PATH=$host_prefix -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_FIND_ROOT_PATH=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release $qtsrcdir
+    ;;
+
+esac
+
+cmake --build . --parallel ${nproc}
+cmake --install .
+install_license $WORKSPACE/srcdir/qt*-src-*/LICENSES/GPL-3.0-only.txt
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(filter(!Sys.isapple, supported_platforms()))
+# Remove when X11 stack will support armv6l and aarch64-freebsd
+filter!(p -> (arch(p) != "armv6l") && !(Sys.isfreebsd(p) && arch(p) == "aarch64"), platforms)
+platforms_macos = [ Platform("x86_64", "macos"), Platform("aarch64", "macos") ]
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct(["Qt6Charts", "libQt6Charts", "QtCharts"], :libqt6charts),
+    LibraryProduct(["Qt6ChartsQml", "libQt6ChartsQml", "QtChartsQml"], :libqt6chartsqml),
+]
+
+products_macos = [
+    FrameworkProduct("QtCharts", :libqt6svg),
```

Comment:
```suggestion
    FrameworkProduct("QtCharts", :libqt6charts),
```
I assume you meant for this product to have the same name as the other platforms.

Reply:
Missed that, thanks!